### PR TITLE
add efficiency cuts to dl2 to irfs options

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

File: lstmcpipe/config/paths_config.py [hunk 1]
```diff
@@ -304,8 +304,10 @@ def path_dict(gamma_part, offset):
                     'electron_file': self.dl2_output_file('electron'),
                 },
                 'output': os.path.join(self.irf_dir(gamma_src_offset=offset), f'irf_{self.prod_id}_{offset}.fits.gz'),
-                'options': '--point-like' if gamma_part == 'gamma' else '',
+                'options': '--gh-efficiency 0.7 --theta-containment 0.7 '
```
Comment (morcuended) [inline lstmcpipe/config/paths_config.py:307]: add "--energy-dependent-gh --energy-dependent-theta" I think by default global cuts are used

File: lstmcpipe/config/paths_config.py [hunk 2]
```diff
@@ -813,18 +815,17 @@ def dl2_to_irfs(self):
...
-                {
+            pp = {
                     'input': {
                         'gamma_file': self.dl2_output_file(pointing),
                         'proton_file': None,
                         'electron_file': None,
                     },
                     'output': os.path.join(self.irf_dir(pointing), f'irf_{self.prod_id}_{pointing}.fits.gz'),
-                    'options': '--point-like',
+                    'options': '--point-like --gh-efficiency 0.7 --theta-containment 0.7 ',
```
Comment (morcuended) [inline lstmcpipe/config/paths_config.py:825]: same as above

**Review by morcuended — CHANGES_REQUESTED** [review]

File: lstmcpipe/config/paths_config.py (hunk 1 above)
Comment (vuillaut) [reply lstmcpipe/config/paths_config.py:307]: right! thanks !

**Review by morcuended — APPROVED** [review]
looks good to me

**Pull request merged**